constants/SeaCreatures.json

```diff
@@ -23,7 +25,7 @@
       "Carrot King": {
         "chat_message": "§aIs this even a fish? It's the Carrot King!",
         "fishing_experience": 1819,
-        "rare": true
```

Comment:
I do not think it is a good idea to remove the "rare" fields as this will stop this feature from working on outdated versions. While it won't crash them it also won't work properly so just have both i guess

Reply:
Good point, all fixed up